scripts/bump_integration.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -euo pipefail
+
+export MANIFEST_PATH="packages/cloud_security_posture/manifest.yml"
+export CHANGELOG_PATH="packages/cloud_security_posture/changelog.yml"
+export INTEGRATION_REPO="orouz/integrations" # TODO: change to elastic/integrations
+export BRANCH="bump-to-$NEXT_CLOUDBEAT_VERSION"
+export MAJOR_MINOR_CLOUDBEAT=$(echo "$NEXT_CLOUDBEAT_VERSION" | cut -d. -f1,2)
+export MAJOR_MINOR_INTEGRATION=$(echo "$NEXT_INTEGRATION_VERSION" | cut -d. -f1,2)
+
+checkout_integration_repo() {
+    gh auth setup-git
+    gh repo clone $INTEGRATION_REPO
+    cd integrations
+    # TODO: use obtained service account user
+    git config --global user.email "elasticmachine@users.noreply.github.com"
+    git config --global user.name "Elastic Machine"
+    git checkout -b "$BRANCH" main
+}
+
+update_manifest_version_vars() {
+    # cis_gcp
+    sed -i'' -E "s/cloudshell_git_branch=[0-9]+\.[0-9]+/cloudshell_git_branch=$MAJOR_MINOR_CLOUDBEAT/g" $MANIFEST_PATH
+
+    # cis_aws + vuln_mgmt_aws
+    sed -i'' -E "s/cloudformation-cnvm-[0-9]+\.[0-9]+\.[0-9]+/cloudformation-cnvm-$NEXT_CLOUDBEAT_VERSION/g" $MANIFEST_PATH
+    sed -i'' -E "s/cloudformation-cspm-ACCOUNT_TYPE-[0-9]+\.[0-9]+\.[0-9]+/cloudformation-cspm-ACCOUNT_TYPE-$NEXT_CLOUDBEAT_VERSION/g" $MANIFEST_PATH
+
+    # cis_azure
+    sed -i'' -E "s/cloudbeat%2F[0-9]+\.[0-9]+/cloudbeat%2F$MAJOR_MINOR_CLOUDBEAT/g" $MANIFEST_PATH
+
+    git add $MANIFEST_PATH
+    git commit -m "Update manifest template vars"
+    git push origin $BRANCH
+}
+
+create_integrations_pr() {
+  echo 'Creating a PR to update integration'
+
+  export PR_URL="$(gh pr create --title "[Cloud Security] Bump integration" \
```

Comment:
Exporting it here to be used later on `update_changelog_version` definitely works. But why not return the value and use as a parameter for `update_changelog_version` function? Seems a bit more clear to me.